darwin/cli.py

```diff
@@ -22,61 +35,78 @@ def main():
 
     # Authenticate user
     if args.command == "authenticate":
-        email = input("Username (email address): ")
-        password = getpass.getpass(prompt="Password: ", stream=None)
-        projects_dir = prompt("Project directory", "~/.darwin/projects")
-        f.authenticate(email, password, projects_dir)
-        print("Authentication succeeded. ")
+        api_key = getpass.getpass(prompt="API key: ", stream=None)
+        if api_key.strip() == "":
+            print(
+                "API Key needed, generate one for your team: https://darwin.v7labs.com/?settings=api-keys"
+            )
+            return
+        default_team = input("Make this team default? [y/N] ") in ["Y", "y"]
+        datasets_dir = prompt("Datasets directory", "~/.darwin/datasets")
+        f.authenticate(api_key, datasets_dir, default_team)
+        print("Authentication succeeded.")
 
     # Select / List team
     elif args.command == "team":
         if args.team_name:
             f.set_team(args.team_name)
-        elif args.list:
-            f.list_teams()
-        else:
+        elif args.current:
             f.current_team()
-
-    # Create new project
-    elif args.command == "create":
-        f.create_dataset(args.project_name)
+        else:
+            f.list_teams()
 
     # List existing projects
     elif args.command == "local":
         f.local()
 
-    # Print projects local path
-    elif args.command == "path":
-        path = f.path(args.project_name)
-        print(path)
-
-    # Print the url of a remote project
-    elif args.command == "url":
-        url = f.url(args.project_name)
-        print(url)
-
     # Download dataset
     elif args.command == "pull":
         project_name = args.project_name
         local_dataset = f.pull_project(project_name)
         print(f"Project {project_name} downloaded at {local_dataset.local_path}. ")
 
-    # List existing projects (remotely)
-    elif args.command == "remote":
-        f.remote()
-
-    # Remove a project (remotely)
-    elif args.command == "remove":
-        project_name = args.project_name
-        f.remove_remote_project(project_name)
-
-    # Upload new data to a project (remotely)
-    elif args.command == "push":
-        f.upload_data(args.project_name, args.files, args.exclude, args.fps)
-
     # Version
     elif args.command == "version":
-        print("0.0.1")
+        print("0.3")
+
+    elif args.command == "dataset":
+        if args.action == "remote":
```

Comment:
What about `list` as a command name instead or remote to match e.g. conda's commands or `ls` to match Docker?

Reply:
the reason we went for remote was that we want to clearly show the difference between datasets on the server vs datasets on the local machine. I don't think docker lets you list remote images.

But another name than remote might be good.